src/schema/__tests__/show.test.js

```diff
@@ -46,11 +46,14 @@ describe("Show type", () => {
 
     rootValue = {
       showLoader: sinon.stub().returns(Promise.resolve(showData)),
-      showsWithHeadersLoader: sinon
```

Comment:
There seem like there are a couple of formatting-only changes here - this might be fine if it's intentional, but worth checking to see if you're applying prettier rule that are overriding project rules. I recently had a big PR that totally reformatted a file in the same vein, had to rewrite that file after fixing my formatting rules?

Reply:
Oooh good catch, let me check my Prettier settings!